<launch>
  <arg name="robot_name" default="/"/>
  
  <group ns="$(arg robot_name)">
    <param name="tf_prefix" value="$(arg robot_name)"/>
    <node pkg="gmapping" type="slam_gmapping" name="gmapping">
      <remap from="scan" to="base_scan"/>
      <remap from="map_metadata" to="local_map_metadata"/>
      <param name="map_frame" value="/$(arg robot_name)/map"/>
      <param name="odom_frame" value="$(arg robot_name)/odom"/>
      <param name="base_frame" value="$(arg robot_name)/base_link"/>
      <param name="xmin" value="-60"/>
      <param name="ymin" value="-60"/>
      <param name="xmax" value="60"/>
      <param name="ymax" value="60"/>
    </node>
  </group>
</launch>
